Comp2024/src/main/java/frc/robot/subsystems/CommandSwerveDrivetrain.java

```diff
@@ -223,7 +223,7 @@ private void startSimThread( )
    */
   private void visionUpdate( )
   {
-    boolean useMegaTag2 = true; //set to false to use MegaTag1
+    boolean useMegaTag2 = false; //set to false to use MegaTag1
```

Comment:
Should this have been checked in? Consult with Grace to see what should be on the main branch (since this will soon be merged). When making your PRs, make sure that all your changes should be a part of your PR.